src/librustdoc/core.rs

```diff
@@ -403,11 +403,23 @@ pub(crate) fn run_global_ctxt(
             {}/rustdoc/how-to-write-documentation.html",
             crate::DOC_RUST_LANG_ORG_VERSION
         );
+
+        let krate_name = if let Some(name) = krate.module.inner.name {
```

Comment:
What's the added value of adding the crate name in the lint since we now have the file path?